pkg/policy/l4.go

```diff
@@ -254,7 +258,7 @@ func (l4 L4PolicyMap) HasRedirect() bool {
 //   constraints that require labels not present in `labels`.
 // Otherwise, returns api.Allowed.
 func (l4 L4PolicyMap) containsAllL3L4(labels labels.LabelArray, ports []*models.Port) api.Decision {
-	if len(l4) == 0 {
+	if len(l4.Filters) == 0 {
```

Comment:
Do we need to also check for `if len(l4) == 0 {`  before checking for l4.filters and return some error if its nil?

Reply:
@manalibhutiyani `l4` is no longer a map.